pwa/app/components/tba/matchResultsTable.tsx

```diff
@@ -110,13 +114,71 @@ function ConditionalTooltip({
   return <>{children}</>;
 }
 
-export default function MatchResultsTableBase({
-  matches,
-  matchTitleFormatter,
-}: {
+interface MatchResultsTableProps {
   matches: Match[];
-  matchTitleFormatter: (match: Match) => string;
-}) {
+  team?: Team;
+  event?: Event;
```

Comment:
make not optional

Reply:
Team is optional here because it should be the same component on a team page as an event page; the team page, however, has the underline highlighting for the team as the only difference